libs/wingsdk/src/util/util.ts

```diff
@@ -22,4 +24,13 @@ export class Util {
   public static tryEnv(name: string): string | undefined {
     return process.env[name];
   }
+
+  /**
+   * Suspends execution for a given duration.
+   *
+   * @param duration The duration that has to pass before the Promise is resolved.
+   */
```

Comment:
I think you'll need to annotate this with:

```suggestion
   * @inflight
   */
```